Installation/CMakeLists.txt

```diff
@@ -1317,12 +1303,12 @@ ${CMAKE_CURRENT_SOURCE_DIR}/../${package}/include/${header}"
             -P "${CGAL_MODULES_DIR}/process_dependencies.cmake"
             ${check_pkg_headers_depends}
           )
-        add_custom_target(${package}_deps DEPENDS ${CGAL_BINARY_DIR}/package_info/${package}/dependencies)
+        add_custom_target(pkg_${package}_deps DEPENDS ${CGAL_BINARY_DIR}/package_info/${package}/dependencies)
         list(APPEND check_pkg_target_list check_pkg_${package}_headers)
         list(APPEND packages_deps ${CGAL_BINARY_DIR}/package_info/${package}/dependencies)
       endif() # if the package has an include directory
-      if(${package}_deps)
-        list(REMOVE_DUPLICATES ${package}_deps)
+      if(packages_deps)
+        list(REMOVE_DUPLICATES packages_deps)
```

Comment:
Does that really work ? Shouldn't it be ${packages_deps} ?

Reply:
For the [if] command, if can test both values or variables:
> `if(<variable>)`
>  if the variable is defined to a value that is not a false constant. False otherwise. (Note macro arguments are not variables.)

[if]: https://cmake.org/cmake/help/v3.10/command/if.html

For the [list] command:
> `list(REMOVE_DUPLICATES <list>)`

That command could not work if you apply it to a list *value*. You have to pass it the list *variable name*, otherwise it cannot modify the variable itself.

[list]: https://cmake.org/cmake/help/v3.10/command/list.html

Reply:
Ok, then.